robottelo/datafactory.py

```diff
@@ -428,3 +428,35 @@ def invalid_http_credentials(url_encoded=False):
                 for cred in credentials]
     else:
         return credentials
+
+
+@datacheck
+def get_valid_preupgrade_data(test_entity, field):
+    """Returns valid list of tuples for given test_entity and field from
+    preUpgrade YAML file
+
+    :param str test_entity: The test_entity name in YAML file
+     e.g. Organization-Tests to check it exists and its associations
```

Comment:
This should be indented. Every following line for the `:param ...:` should be indented otherwise it will break documentation formatting.